packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import '../_elementPolyfill';
+import * as KeyCode from 'keycode-js';
+import styles from '../WorklistDataGrid.module.scss';
```

Comment:
You should have a separate sass file for the cell and other components that were created.  We would not want to place them all in the WorklistDataGrid sass file.